content/references.bib

```diff
@@ -302,24 +303,22 @@ @article{viezzoli2009
 number = {2},
 pages = {173-183},
 year = {2009},
-doi = {10.1071/EG08027},
-URL = {
-        http://library.seg.org/doi/abs/10.1071/EG08027
-    },
-}
```

Comment:
@dccowan: I believe that the failure is due to the deleted lines above (305 - 309). Without the closed bracket here, the references file can't be parsed correctly.